experimental/packages/api-logs/src/internal/global-utils.ts

```diff
@@ -14,12 +14,10 @@
  * limitations under the License.
  */
 
-import { LoggerProvider } from '../types/LoggerProvider';
-import { _globalThis } from '../platform';
+import type { LoggerProvider } from "../types/LoggerProvider";
+import { _globalThis } from "../platform";
```

Comment:
There are many places you decided to change just the quote for some reason. This strikes me as an autoformatter that you ran, but it really bloats the PR

Reply:
Sorry, I took the time to do some formatting. If it is not needed, I will delete it here

Reply:
Formatter enforcement has been enabled in https://github.com/open-telemetry/opentelemetry-js/pull/3444. Now the styles can be validated with GitHub Actions.

Reply:
Ok, that's great. I'm just going to restore the format here